tools/markdown-magic/src/templates/Client-Requirements-Template.md

```diff
@@ -9,6 +9,7 @@ When making such a request please include if the configuration already works (an
 # Supported Runtimes
 
 -   NodeJs ^20.10.0 except that we will drop support for it [when NodeJs 20 loses its upstream support on 2026-04-30](https://github.com/nodejs/release#release-schedule), and will support a newer LTS version of NodeJS (22) at least 1 year before 20 is end-of-life. This same policy applies to NodeJS 22 when it is end of life (2027-04-30).
+-   Running Fluid in a Node.js environment with the `--no-experimental-fetch` flag is no longer supported.
```

Comment:
I think the indentation to make it a sub-item of the main Node bullet point is preferrable. Also, since this bit of content isn't "timestamped" or tied to a particular release in this file and the files it touches, I think present tense fits better.
```suggestion
  -   Running Fluid in a Node.js environment with the `--no-experimental-fetch` flag is not supported.
```